packages/blocks/src/page-block/utils/container-operations.ts

```diff
@@ -74,6 +75,28 @@ export function deleteModels(page: Page, models: BaseBlockModel[]) {
   firstRichText.quill.setSelection(firstTextIndex, 0);
 }
 
+function mergeTextOfBlocks(
+  page: Page,
+  models: BaseBlockModel[],
+  blockProps: Record<string, unknown>
+) {
+  const parent = page.getParent(models[0]);
+  assertExists(parent);
+  const index = parent.children.indexOf(models[0]);
+  const id = page.addBlock(blockProps, parent, index);
+  const codeBlock = page.getBlockById(id) as CodeBlockModel;
+
+  models.forEach(model => {
+    if (model.text instanceof Text) {
```

Comment:
Maybe we need further discussion about just skipping other blocks...

Reply:
OK! Not converting some type of blocks also make sense